src/main/java/com/sprint/mission/discodeit/service/basic/BasicReadStatusService.java

```diff
@@ -0,0 +1,77 @@
+package com.sprint.mission.discodeit.service.basic;
+
+import com.sprint.mission.discodeit.dto.readStatus.ReadStatusCreateRequest;
+import com.sprint.mission.discodeit.dto.readStatus.ReadStatusUpdateRequest;
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+import com.sprint.mission.discodeit.repository.ReadStatusRepository;
+import com.sprint.mission.discodeit.repository.UserRepository;
+import com.sprint.mission.discodeit.service.ReadStatusService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class BasicReadStatusService implements ReadStatusService {
+    private final ReadStatusRepository readStatusRepository;
+    private final UserRepository userRepository;
+    private final ChannelRepository channelRepository;
+
+    @Override
+    public ReadStatus create(ReadStatusCreateRequest request) {
+        User user = userRepository.findByUserId(request.userId());
+        if (user == null) {
+            throw new IllegalArgumentException("사용자를 찾을 수 없습니다: " + request.userId());
+        }
+
+        Channel channel = channelRepository.findByChannelId(request.channelId());
+        if (channel == null) {
+            throw new IllegalArgumentException("채널을 찾을 수 없습니다: " + request.channelId());
+        }
+
+        List<ReadStatus> userReadStatuses = readStatusRepository.findAllByUserId(request.userId());
+        for (int i = 0; i < userReadStatuses.size(); i++) {
+            ReadStatus status = userReadStatuses.get(i);
+            if (status.getChannelId().equals(request.channelId())) {
```

Comment:
향상된 for문을 사용하면 코드가 훨씬 간결해질 것 같네요.

```suggestion
        for (ReadStatus status : userReadStatuses) {
            if (status.getChannelId().equals(request.channelId())) {
```